eng/codegen_to_sdk_config.json

```diff
@@ -1,31 +0,0 @@
-{
```

Comment:
@raych1 can you confirm we don't use these any longer? I see we have similar files in the other languages https://github.com/search?q=org%3AAzure+codegen_to_sdk_config&type=code

Reply:
This file was previously used by the docker-based SDK generation tool, which had been deleted.